mainhandler/handlerequests.go

```diff
@@ -115,8 +136,16 @@ func (mainHandler *MainHandler) SetupContinuousScanning(ctx context.Context) err
 	}
 
 	fetcher := cs.NewFileFetcher(rulesReader)
+	if fetcher == nil {
+		panic("fetcher is nil")
```

Comment:
We should return the error and handle it in the calling function.

Reply:
Now _that_ is a leftover from debugging. Will fix.

Reply:
Fixed.